src/python/pants/backend/docker/target_types.py

```diff
@@ -169,21 +169,30 @@ class DockerBuildSecretsOptionField(
     AsyncFieldMixin, DockerBuildOptionFieldMixin, DictStringToStringField
 ):
     alias = "secrets"
-    help = "Secret file to expose to the build (only if BuildKit enabled).\n\n" + dedent(
-        """\
-        Example:
-
-            docker_image(
-                secrets={
-                    "mysecret": "/local/secret",
-                }
-            )
-        """
+    help = (
+        "Secret file to expose to the build (only if BuildKit enabled).\n\n"
+        "Secrets may use absolute paths, or paths relative to your BUILD file. The id should be "
+        "valid as used by the Docker build `--secret` option. "
+        "See [Docker secrets](https://docs.docker.com/engine/swarm/secrets/) for more "
+        "information.\n\n"
+        + dedent(
+            """\
+            Example:
+
+                docker_image(
+                    secrets={
+                        "mysecret": "/local/secret",
```

Comment:
`/local` is confusing imo because it makes me think of a relative path. Maybe instead use `/var/secret`? And then bump on having a second example with `rel_path`

Reply:
I feel I can save this for a later change, ok?

It's not code to be read a lot, by others than maintainers/contributors, and is not critical. The path begins with `/` so to me is clearly absolute, but I'm not opposed to change `local => var` to make it even more clear. Ditto on working with the rel path wording to make that clearer as well.

> And then bump on having a second example with `rel_path`

Do you mean splitting this into two separate tests? That to me would feel a little bit overkill, and added cost in terms of test code noise and duplication so, little to be gained by that imo.

Reply:
This is the help message, so I think it's read by everyone. Agreed if it was the test that it doesn't matter much

Not a huge deal if you want to land

Reply:
Oh, right. Haha, ok. Yeah, would prefer to fix in a follow up. Thought it was the test code. My bad ;)

Reply:
That's fine with me. Thanks for the PR!